### ISSUE TITLE
Nosecounter: H tags used for presentation

### ISSUE THREAD

**Issue by Burfdl** [eurofurence/website#38]
[WCAG SC 1.6.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) suggests [H42: Using h1-h6 to identify headings](https://www.w3.org/WAI/WCAG21/Techniques/html/H42) and calls out [using h1-6 for visual or presentation effect as failures of 1.6.1 under F43](https://www.w3.org/WAI/WCAG22/Techniques/failures/F43)

on the nose counter page:

<img width="457" height="245" alt="Image" src="https://github.com/user-attachments/assets/9f30f79f-8cd6-4a94-9075-df25f4d33bd9" />

`<h2>` is used to adjust margins on total registrations number, and

<img width="440" height="163" alt="Image" src="https://github.com/user-attachments/assets/5fbfc2b2-6861-4d8a-88e9-9b366757b8d9" />

`<h4>` is used in the country list to make the country codes bold.

**Comment by dogpixels** [comment]
Very well documented. Thank you once again for your valuable contribution!
